src/js/binaryen.js-post.js

```diff
@@ -1049,10 +1049,10 @@ function wrapModule(module, self = {}) {
     'store32'(offset, align, ptr, value, name) {
       return Module['_BinaryenStore'](module, 4, offset, align, ptr, value, Module['i64'], strToStack(name));
     },
-    'const'(x, y) {
+    'const'(x) {
       return preserveStack(() => {
         const tempLiteral = stackAlloc(sizeOfLiteral);
-        Module['_BinaryenLiteralInt64'](tempLiteral, x, y);
+        Module['_BinaryenLiteralInt64'](tempLiteral, BigInt(x));
```

Comment:
What do you think about making this backwards compatible? We could support `(i32, undefined), (i32, i32), and (BigInt, undefined)` with this code:
```js
        // Convert to a BigInt. We support (x, undefined) where x is either a
        // BigInt or not, and also (x, y) where neither is a BigInt and y is the
        // higher bits.
        if (typeof x !== 'bigint') {
          x = BigInt(x || 0) | (BigInt(y || 0) << 32n);
          console.log('big', x);
        }
```

Reply:
A small code size cost seems reasonable to me here, personally. And it is nice to write `binaryen.i64.const(42)` and not need to remember `42n` each time.

Reply:
`binaryen.i64.const(42)` already works because of the explicit `BigInt(x)` conversion here. Except for compatibility, I don't think there's much benefit to supporting the two-argument version. How much do we value compatibility here? Would we have a deprecation plan, or would we support the two-argument version indefinitely?

Reply:
I think compatibility makes sense: there are users that are creating i64s with two arguments, and I don't see a strong reason to break them, given the burden on supporting the old format is minimal.